packages/realm-web/src/index.ts

```diff
@@ -38,26 +36,11 @@ export function app(id: string) {
     }
 }
 
-/**
- * Handle an OAuth 2.0 redirect.
- *
- * @param location An optional location to use (defaults to the windows current location).
- * @param storage Optional storage used to save any results from the location.
- */
-export function handleAuthRedirect(
-    location = window.location,
-    storage = createDefaultStorage(),
-) {
-    try {
-        const queryString = location.hash.substr(1); // Strip the initial # from the hash
-        OAuth2Helper.handleRedirect(queryString, storage);
-    } catch (err) {
-        // Ensure calling this never throws: It should not interrupt a users flow.
-        console.warn(err);
-    }
-}
-
 export { App };
 export { Credentials } from "./Credentials";
 export { User, UserState } from "./User";
-export { createDefaultStorage } from "./storage";
+export * from "./environment";
```

Comment:
Isn't the environment stuff internal implementation details? Do we really want to be exporting it like this?

Reply:
It's exposed primarily for [the integration tests to hook into](https://github.com/realm/realm-js/blob/kh/realm-web-iife/packages/realm-web-integration-tests/src/app.test.ts#L40). I don't think we'll need to include this in any generated API docs.

Reply:
Maybe I don't quite understand how this works exactly. When someone imports realm-web into a TS-using project, would they ever see the "real" types that you are using here, or would they only see the "fake" ones from the realm-js/types directory?